plonky2x/src/utils/mod.rs

```diff
@@ -46,6 +46,20 @@ pub fn byte_to_bits_be(input: u8) -> [bool; 8] {
     bits
 }
 
+pub fn to_be_bits(msg: &[u8]) -> Vec<bool> {
```

Comment:
I think it will be helpful to leave a comment about the input's and output's bit-endianness.

Reply:
nvm, just noticed that the function name has that info.